nav2_gps_waypoint_follower_demo/config/nav2_no_map_params.yaml

```diff
@@ -0,0 +1,307 @@
+bt_navigator:
```

Comment:
Question: is there a reason this can't be using the default file? What are the differences in this file? 

Perhaps a comment block with the nodes that have any changes would be helpful for those scanning to find the configurations that differ from the main file. Also, it would help me as a maintainer to keep this in sync if I knew what nodes had no changes to just quash and overwrite without worry

Reply:
key differences are
1. Complete removal of amcl
2. static layer removed from the costmaps 
3. Global costmap is set to be rolling with a fixed size

Reply:
added some general comments at the beginning of the file and comments referring to the specific changes above each node's section